dev/com.ibm.ws.microprofile.rest.client.FT_fat/test-applications/retryApp/src/mpRestClientFT/retry/RetryTestServlet.java

```diff
@@ -63,6 +59,7 @@ public void testRetryOnceOnFailThenSucceed_separateMethod(HttpServletRequest req
     }
 
     @Test
+    @SkipForRepeat(JakartaEE9Action.ID) // CDI not resolving app-supplied interceptor here TODO: investigate and resolve
```

Comment:
is this a big deal? is there an issue open?

Reply:
Open issue - possibly related to the beans.xml issue you pointed out earlier??

Reply:
please can you add a link to the open issue? here and/or in the code